README.md

```diff
@@ -1,100 +0,0 @@
-## DGL Implementation of the MixHopGCN Paper
-
-This DGL example implements the GNN model proposed in the paper [MixHop: Higher-Order Graph Convolution Architectures via Sparsified Neighborhood Mixing](https://arxiv.org/abs/1905.00067). The author's codes of implementation is in [here](https://github.com/samihaija/mixhop).
```

Comment:
"The author's codes of implementation is in [here](https://github.com/samihaija/mixhop)." -> "For the original implementation, see [here](https://github.com/samihaija/mixhop)."

Reply:
Done.